Ginger/Ginger/ALM/Repository/ALMRepository.cs

```diff
@@ -55,6 +56,10 @@ public void SetALMProject(string project)
         public abstract eUserMsgKeys GetDownloadPossibleValuesMessage();
         public abstract IEnumerable<Object> SelectALMTestSets();
         public abstract bool ImportSelectedTests(string importDestinationPath, IEnumerable<Object> selectedTests);
+        public abstract List<string> GetTestLabExplorer(string path);
+        public abstract List<QCTestSetSummary> GetTestSetExplorer(string path);
+        public abstract QCTestSetSummary GetTSRunStatus(QCTestSetSummary tsItem);
+        public abstract List<string> GetTestPlanExplorer(string path);
         public abstract bool ShowImportReviewPage(string importDestinationPath, object selectedTestPlan = null);
```

Comment:
Thise "QCTestSetSummary" list of objects do not seems to be Generic as required for ALMRepository, those seems specific for QC

Reply:
You asked to do the get test set and test plan explorer generic in the QCTestLabExplorerPage and QCTestPlanExplorerPage. In order to do it i need to declare it in the ALMRepository level.

Reply:
Replace List<QCTestSetSummary> with List<Object> and Cast it to QCTestSetSummary in the QC related pages or were needed so ALMRepository will remain generic and not QC specific